WooCommerce/Classes/ViewRelated/Coupons/EnableAnalytics/EnableAnalyticsView.swift

```diff
@@ -0,0 +1,166 @@
+import SwiftUI
+
+/// View to enable WC Analytics for the current store
+///
+struct EnableAnalyticsView: View {
+    @ObservedObject private var viewModel: EnableAnalyticsViewModel
+    @Environment(\.presentationMode) private var presentation
+
+    /// Action to trigger when Contact Support is selected
+    private let contactSupportAction: () -> Void
+
+    /// Closure to trigger when enabling analytics succeeds
+    private let completionHandler: () -> Void
+
+    /// The presenter to display notice when enabling analytics succeeds or fails.
+    private let noticePresenter: DefaultNoticePresenter
+
+    /// Keeping a reference to the presenting controller to present notice and contact support
+    private let presentingController: UIViewController?
+
+    /// The attributed string to display for the contact support button
+    /// Essentially is a message with the Contact Support text highlighted
+    private var contactSupportAttributedString: NSAttributedString {
```

Comment:
I don't think we cannot use a lazy property since accessing it mutates the struct 🤔 I read [somewhere](https://oleb.net/blog/2015/12/lazy-properties-in-structs-swift/) that there's a workaround for that but I don't think the complication is worth it in this case.